rest-model/src/main/java/org/jboss/pnc/rest/restmodel/BuildExecutionConfigurationRest.java

```diff
@@ -95,6 +116,20 @@ public BuildExecutionConfiguration toBuildExecutionConfiguration() {
         );
     }
 
+    public BuildConfigurationAudited toBuildConfigurationAudited() {
+        BuildConfigurationAudited buildConfigAudited = new BuildConfigurationAudited();
```

Comment:
This object was not meant to be directly mapped to DB entities and a requirement is to make rest-model independent of DB model.